api/models.py

```diff
@@ -5,10 +5,10 @@
 class UserProfile(models.Model):
 
 	ACTIVITY_LEVEL = (
-			("SC", "Sleepy_Cat"),
-			("LC", "Lazy_Cat"),
-			("AC", "Active_Cat"),
-			("MC", "Mentor_Cat"),
+			("04", "Sleepy_Cat"),
```

Comment:
why the changes to the model?
these changes would require a change in migrations, but there doesn't seem to be one.

I recommend these changes to be reverted